.gitmodules

```diff
@@ -86,3 +86,12 @@
 	path = third_party/wifi-ism43362/repo
 	url = https://github.com/ATmobica/wifi-ism43362.git
 	branch = master
+[submodule "third_party/repo"]
+	path = third_party/repo
+	url = https://github.com/Infineon/capsense.git
+[submodule "third_party/third_party/capsense/repo"]
+	path = third_party/third_party/capsense/repo
+	url = https://github.com/Infineon/capsense.git
+[submodule "third_party/capsense/repo"]
+	path = third_party/capsense/repo
+	url = https://github.com/Infineon/capsense.git
```

Comment:
what happened here? Why are there so many new submodules?

Reply:
Mistake. To be removed